internal/storage/migrations/14-create-psu-tables.sql

```diff
@@ -27,21 +27,11 @@ create table if not exists payment_service_users (
 
 create index psu_created_at_sort_id on payment_service_users (created_at, sort_id);
 
-create table if not exists psu_bank_accounts (
-    -- Mandatory fields
-    psu_id uuid not null,
-    bank_account_id uuid not null,
-
-    primary key (psu_id, bank_account_id)
-);
+alter table bank_accounts 
+    add column if not exists psu_id uuid default null;
```

Comment:
I think you'll get the same effect without aggressively locking the entire table if you change the alter table to this:
```
add column if not exists psu_id uuid;
```

Not defining a default means that value will be null by default. But I think the moment you mention the `DEFAULT` keyword in an alter table postgres will choose an access exclusive lock.